numpy/core/tests/test_multiarray.py

```diff
@@ -2758,11 +2758,6 @@ def test_flatten(self):
         assert_equal(x1.flatten('F'), y1f)
         assert_equal(x1.flatten('F'), x1.T.flatten())
 
-    def test_flatten_invalid_order(self):
-        # invalid after gh-14596
-        for order in ['Z', 'c', False, True, 0, 8]:
-            x = np.array([[1, 2, 3], [4, 5, 6]], np.int32)
-            assert_raises(ValueError, x.flatten, {"order": order})
```

Comment:
This test was actually calling `x.flatten(dict(order=...))`, which fails no matter what `...` is.